tests/test_mailtrap_webhooks.py

```diff
@@ -0,0 +1,375 @@
+import json
+from datetime import datetime, timezone
+from unittest.mock import ANY
+
+from django.test import tag
+
+from anymail.signals import AnymailTrackingEvent
+from anymail.webhooks.mailtrap import MailtrapTrackingWebhookView
+
+from .webhook_cases import WebhookBasicAuthTestCase, WebhookTestCase
+
+
+@tag("mailtrap")
+class MailtrapWebhookSecurityTestCase(WebhookBasicAuthTestCase):
+    def call_webhook(self):
+        return self.client.post(
+            "/anymail/mailtrap/tracking/",
+            content_type="application/json",
+            data=json.dumps({}),
+        )
+
+    # Actual tests are in WebhookBasicAuthTestCase
+
+
+@tag("mailtrap")
+class MailtrapDeliveryTestCase(WebhookTestCase):
+    def test_sent_event(self):
+        payload = {
+            "events": [
+                {
+                    "event": "delivery",
+                    "timestamp": 1498093527,
+                    "sending_stream": "transactional",
+                    "category": "password-reset",
+                    "custom_variables": {"variable_a": "value", "variable_b": "value2"},
+                    "message_id": "1df37d17-0286-4d8b-8edf-bc4ec5be86e6",
+                    "email": "receiver@example.com",
+                    "event_id": "bede7236-2284-43d6-a953-1fdcafd0fdbc",
+                },
+            ]
+        }
+        response = self.client.post(
+            "/anymail/mailtrap/tracking/",
+            content_type="application/json",
+            data=json.dumps(payload),
+        )
```

Comment:
```suggestion
        response = self.client.post(
            "/anymail/mailtrap/tracking/",
            content_type="application/json",
            data=payload,
        )
```

(See earlier note. Same also applies to other post() calls below.)